datanode/gateway/graphql/schema.graphql

```diff
@@ -542,6 +542,34 @@ type Query {
     withdrawalId: ID!
   ): Erc20WithdrawalApproval
 
+  erc20MultiSigSignerAddedBundles(
+    "The node id of the validator of which a signature bundle is required"
+    nodeId: ID!
+    "The ethereum address of the submitter"
+    submitter: String
+    "The epoch which generated the bundle i.e the epoch in which the node was demoted from a tendermint validator"
```

Comment:
```suggestion
    "The epoch that generated the bundle i.e the epoch in which the node was demoted from a tendermint validator"
```

Reply:
Should this be "the epoch in which the node was promoted to tendermint validator" if it's under `MultiSigSignerAdded`?

Reply:
Honestly I don't really understand much of what these are all about, I've shamelessly copy-pasted these comments from the `.proto` file.

Reply:
It should be promoted (my fault, I added the protos). Its probably worth fixing the protos comments too while we're here.